i2c/lcd_1602_i2c/main.c

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2025 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include "i2c_lcd.h"
+#include "pico/stdlib.h"
+#include "hardware/gpio.h"
+#include "hardware/i2c.h"
+
+/*
+ * Example code to drive a 16x2 LCD panel via a I2C bridge chip (e.g. PCF8574)
+ * NOTE: The panel must be capable of being driven at 3.3v NOT 5v. The RP2040
+ * is not 5V tolerant.
+ *
+ * You will need to use a level shifter on the I2C lines if you want to run the
+ * board at 5V. Using it without a level shifter will work but may damage the I/O
+ * cells on the RP2040 over time since the I2C pullups on the display will pull
+ * the data lines up to 5V.
+ *
+ * Connections on Raspberry Pi Pico board, other boards may vary.
+ * GPIO 0 -> SDA on LCD bridge board
+ * GPIO 1 -> SCL on LCD bridge board
+ */
+
+// You may need to change this for your LCD module
+#define I2_LCD_ADDR 0x27
+
+int main()
+{
+	// Set up USB virtual COM port
+	stdio_init_all();
+
+	// Set up the I2C peripheral at 100k
+	i2c_init(i2c0, 100 * 1000);
+
+	// Bind pins to I2C peripheral
+	gpio_set_function(0, GPIO_FUNC_I2C);
```

Comment:
Rather than hard-coding this to GPIO 0 and GPIO 1, could you use `PICO_DEFAULT_I2C_SDA_PIN` and `PICO_DEFAULT_I2C_SCL_PIN` please, as the original example did? That way this example will still build and run on boards that make different I2C pins accessible to the user (see https://github.com/raspberrypi/pico-sdk/tree/develop/src/boards/include/boards for examples of this being used).

Reply:
Fixed